### PR TITLE
Docs fixes

### PR DESCRIPTION
**Pull request by theonlyjohnny**
I'm not sure where `leaderboard.min.js` is hosted, so the docs have `src=""` for now.

### CODE REVIEW

File: docs/api_methods.rst [hunk 1]
```diff
@@ -3,10 +3,11 @@ Overview
...
-* :ref:`database-api`
-* :ref:`notifications-api`
-* :ref:`camera-api`
-* :ref:`misc-api`
\ No newline at end of file
+* `User Api <user_api.html>`_ 
+* `Room Api <room_api.html>`_
+* `Server Api <server_api.html>`_
+* `Database Api <database_api.html>`_
+* `Notifications Api <notification_api.html>`_
```
Comment (jakeloo) [inline docs/api_methods.rst]: nit: plural - singular mismatch.

File: docs/audio_api.rst [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+============
+
+`Bebo.Audio` is for accessing the audio functionality, broadcasting audio on/off the room.
+
+Bebo.Audio.callin
+------------------
+
+`Bebo.Audio.callin();`
+
+The callin() function turns on the users camera and microphone and broadcasts to everyone within your app::
```
Comment (jakeloo) [inline docs/audio_api.rst]: above mention `the room`, and this one `within your app`? inconsistency?

Comment (jakeloo) [reply docs/audio_api.rst]: "your app" may be confusing as in native app, imo.

File: docs/camera_api.rst [hunk 3]
```diff
@@ -21,11 +21,11 @@ The previewOff() function will turn off the native camera preview::
 
     Bebo.previewOff();
 
-Bebo.Camera.capture
+Bebo.Camera.capturePhoto
```
Comment (jakeloo) [inline docs/camera_api.rst:24]: @theonlyjohnny can you add in more info here, 
what do i need to do to get the captured photo.

File: docs/audio_api.rst (hunk 2 above)
Comment (theonlyjohnny) [reply docs/audio_api.rst]: What is the scope for this function? Is it group & room sorted or just room?

File: docs/index.rst [hunk 4]
```diff
@@ -51,13 +51,15 @@ Download `Bebo for iOS <https://itunes.apple.com/us/app/bebo-private-server-for-
    database_api
    notification_api
    camera_api
+   audio_api
    misc_api
 
 .. toctree::
    :maxdepth: 2
-   :caption: Libraries
+   :caption: Modules
```
Comment (thetylerhayes) [inline docs/index.rst]: the react lib is a lib, not a module. either create a new section called Modules (i'd suggest renaming that to something more conventional; hard to understand what it means right away as a newcomer), or put the leaderboard module somewhere else in the docs. but don't hide the react lib in an obscure section called modules

File: docs/leaderboard.rst [hunk 5]
```diff
@@ -0,0 +1,120 @@
+Leaderboard
+=====================
+
+
+A Bebo component to quickly add a leaderboard into your HTML5 game. **NOTE: Leaderboard is scoped to group and app, so the leaderboard from one game won't interfere with another, and won't interfere with the same game on another group.***
```
Comment (thetylerhayes) [inline docs/leaderboard.rst]: use proper `NOTE` reStructuredText notation here. see http://www.sphinx-doc.org/en/stable/rest.html for documentation and our own quick_start.rst and getting_started.rst for examples

File: docs/misc_api.rst [hunk 6]
```diff
@@ -118,4 +90,15 @@ The getStream() function is self explanatory for us but will be broken down into
...
-    });
\ No newline at end of file
+    });
+
+Bebo.openURI
+------------------
+
+`Bebo.openURI(uri)`
+
+The openURI() function will open the passed in URI. This is helpful for sharing, such as to Facebook, Twitter, through What's App, or even through your native SMS client. **NOTE: iOS and Android treat SMS URI's differently, do your research**:
```
Comment (thetylerhayes) [inline docs/misc_api.rst]: same thing applies here re: `NOTE` notation

**Review by thetylerhayes — CHANGES_REQUESTED** [review]
a couple more little changes. overall looks great! thanks for doing these

File: docs/camera_api.rst [hunk 7]
```diff
@@ -24,8 +24,14 @@ The previewOff() function will turn off the native camera preview::
...
 The capture() function will capture a photo if the camera preview is open::
 
-    Bebo.Camera.capturePhoto();
\ No newline at end of file
+    Bebo.Camera.capturePhoto(function(err,url){
+    	if (err){
+    		console.error(err);
+    		return;
+    	}
+    	console.log(url); //<--- url is a url to your image.
```
Comment (jakeloo) [inline docs/camera_api.rst:36]: should be clarified that it is a local image url that only works on the device. 
if it needs to be displayed everywhere, it should be uploaded to image service.
as we talked about in slack :)

File: docs/api_methods.rst (hunk 1 above)
Comment (theonlyjohnny) [reply docs/api_methods.rst]: Fixed

File: docs/camera_api.rst (hunk 3 above)
Comment (theonlyjohnny) [reply docs/camera_api.rst:24]: Fixed

File: docs/index.rst (hunk 4 above)
Comment (theonlyjohnny) [reply docs/index.rst]: Changed back to "Libraries" and added "Add-ons" section (hopefully more useful leaderboard-type projects will be made in the future?)

**Pull request merged**